resources/shaders/accumulation.hlsl

```diff
@@ -16,8 +16,12 @@ void main(uint3 DTid : SV_DispatchThreadID)
 {
 	float2 resolution;
 	input.GetDimensions(resolution.x, resolution.y);
-	
-	float3 color = input[DTid.xy].rgb / (frame_idx + 1);
+
```

Comment:
incorrect merge?

Reply:
This was the latest from master, I changed it to use SampleLevel instead.